src/magic.cpp

```diff
@@ -8214,6 +8273,11 @@ int cast_power_heal( ubyte level, CHAR_DATA *ch, char *arg, int type,
 	 case SPELL_TYPE_POTION:
 	 return spell_power_heal(level, ch, ch, 0, skill);
 	 break;
+	 case SPELL_TYPE_SCROLL:
+	 for (tar_ch = world[ch->in_room].people ;
```

Comment:
Why are we looping through the room for a scroll?  Only staves are typically room-effect for spells. Is this intended?